lib/ffmpeg/transcoder.rb

```diff
@@ -36,16 +36,17 @@ def initialize(movie, output_file, options = EncodingOptions.new, transcoder_opt
       if requires_pre_encode
         @movie.paths.each do |path|
           # Make the interim path folder if it doesn't exist
-          dirname = "#{File.dirname(path)}/interim"
+
+          dirname = "/tmp/rlovelett/#{File.dirname(path)}/interim"
```

Comment:
Nit that I'd suggest swapping the dirname & interim order
```suggestion
          dirname = "/tmp/rlovelett/interim/#{File.dirname(path)}/"
```